_posts/2022-10-07-101-so-youre-a-team-supervisor.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Robots 101 - So you're running a team
+---
+
+{% include figure.html src="/images/content/blog/robots-101/team-supervisor.jpg"
+           caption="One of our SR2022 Team Supervisors" %}
+
+So, you’re running a team. Here’s a little bit of info on what to expect from Student Robotics, and your responsibilities as a team supervisor.
+
+As a team supervisor, your role is to guide the competitors through the journey of building a robot. You’ll be there to point them in the right direction when they get stuck and resolve any issues they run into. We encourage team supervisors to limit their involvement with the robot design/building process so that the finished contraptions are 100% student-built which competitors find very rewarding.
+
+You’ll be our point of contact with the team. If you have any questions during the year, just email [teams@studentrobotics.org](mailto:teams@studentrobotics.org). We’ll also be sending you emails every month or so with important information such as:
+- Details of Tech Days where we provide a space for teams to come together and get direct help from our dedicated volunteers
+- Software updates for our kit
+- Information about the competition in April
+
+We aim to host our Kickstart event and Tech Days in multiple locations to make it more convenient for you to travel. However, you will still need to arrange to travel to these places.
```

Comment:
I suppose hosting kickstarts in multiple locations may be a bit out of date at this point, not that it matters too much